packages/rn-tester/Podfile

```diff
@@ -70,23 +70,6 @@ target 'RNTester-macOSIntegrationTests' do
   pod 'React-RCTTest', :path => "./RCTTest"
 end
 
-# [TODO(macOS GH#774): these are special targets used by the internal Microsoft build pipeline
-target 'iosDeviceBuild' do
```

Comment:
Are these not necessary so SDX-Platform can build them later when we pull in rn-macos?

Reply:
SDX-Platform has it's own Podfile in `packages/react-native-community/apple`. In effect, SDX-Platform is the one in charge of turning this public NPM package into a NuGet. I wonder if Android can do the same.

That being said, I expect I could be totally wrong so I'm prepared to back this out if that's the case 😅

Reply:
Hey did you get more insight into these? I remember we needed them for CI builds at some point but agree they might be dated now.